models/intermediate/job/int_greenhouse__job_info.sql

```diff
@@ -31,18 +35,29 @@ final as (
         hiring_team.recruiters,
         hiring_team.coordinators,
 
+        {% if var('greenhouse_using_job_office', True) %}
         job_office.offices,
         job_office.locations as office_locations,
+        {% endif %}
+
+        {% if var('greenhouse_using_job_department', True) %}
         job_department.departments,
         job_department.parent_departments
+        {% endif %}
```

Comment:
This fails in Redshift and Snowflake with the trailing comma if both `greenhouse_using_job_office` and `greenhouse_using_job_department` are false.
```suggestion
        hiring_team.coordinators

        {% if var('greenhouse_using_job_office', True) %}
        ,job_office.offices
        ,job_office.locations as office_locations
        {% endif %}

        {% if var('greenhouse_using_job_department', True) %}
        ,job_department.departments
        ,job_department.parent_departments
        {% endif %}
```